package com.revature.cats.exceptions;

public class UserNotFoundException extends RuntimeException {
	
}
